src/app/store.ts

```diff
@@ -42,14 +42,24 @@ import { pagesSlice } from "../features/Pages/pagesSlice";
 import mergeInitialState from "redux-persist/lib/stateReconciler/autoMergeLevel2";
 import { listenerMiddleware } from "./middleware";
 
+const tipOfTheDayPersistConfig = {
```

Comment:
Is there a reason to have two persisted keys ?

Reply:
In the case of Oleg, his history was 5 megabytes. So every time the tip of the day was changed (when opening a new chat), it would also restore the whole history in local storage. Which was slowing down the chat a bit.